rebuild/config/_icons.scss

```diff
@@ -0,0 +1,152 @@
+.ap-icon {
+    $self: &; // Notes: https: //css-tricks.com/using-sass-control-scope-bem-naming/
+
+    //Icon sizes
+    $icon-sizes: ('tiny': 16px, 'small': 24px, 'medium': 32px, 'ui': 40px, 'large': 88px, 'maxi': 152px);
+
+    //Icons and their file locations
+    $icons: (
+        'hamburger--primary': (
+            'asset': 'rebuild/icons/hamburger--primary.svg',
```

Comment:
>Probably will need to update the urls within /rebuild/config/_icons.scss I'm not show how these will be pulled in from node_modules? Or is this something Lerna will do for us?

Won't be solved by Lerna in this instance. If we replace `rebuild` with `$image-dir`, we can set it in the relevant stylesheet. I think for the new site, we'll gulp them into our asset directory (httpdocs) on deploy and run them through an optimiser, but on the legacy site for the time being I think we'll probably load them straight from node-modules

Reply:
Added variable for location, defaults to node modules. Let me know if we should use a different URL for default.

Reply:
Moved icons from `rebuild/icons/` to `rebuild/assets/images/icons` incase we add any new files to the assets folder in the future. It also makes more sense for the `$image-dir` variable to point to the `rebuild/assets/images/` folder.